app/helpers/sessions_helper.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+module SessionsHelper
+  def sign_in(user)
+    session[:user_id] = user.id
+  end
+
+  def current_user
+    @current_user ||= User.find_by(id: session[:user_id]) if session[:user_id]
+  end
+
+  def signed_in?
+    !current_user.nil?
+  end
+
+  def sign_out
+    session.delete(:user_id) if signed_in?
+    @current_user = nil
+  end
+end
```

Comment:
The idea of helpers is to use them specifically for views if you are using some of those helper methods in controllers or models it is better to move them to the `/lib` folder or the `ApplicationController`. Please check this useful link: http://codefol.io/posts/Where-Do-I-Put-My-Code/

Reply:
![image](https://user-images.githubusercontent.com/29500109/69881839-39963480-1309-11ea-9c79-c106177dad70.png)

Helper codes have been moved to application controller